ports/cef/window.rs

```diff
@@ -242,6 +243,51 @@ impl WindowMethods for Window {
         (Size2D::new(width, height), Point2D::zero())
     }
 
+    fn screen_size(&self, _: BrowserId) -> Size2D<u32> {
+        let browser = self.cef_browser.borrow();
+        match *browser {
+            None => Size2D::zero(),
+            Some(ref browser) => {
+                let mut screen_info = cef_screen_info_t {
+                    device_scale_factor: 0 as f32,
+                    depth: 0 as i32,
+                    depth_per_component: 0 as i32,
+                    is_monochrome: 0 as i32,
+                    rect: cef_rect_t::zero(),
+                    available_rect: cef_rect_t::zero(),
+                };
```

Comment:
Not very nice having to do this. Is there an implementation of `Default::default()` we could use to make this slightly better?